### PR TITLE
[ACS][CallAutomation] Update python-packages.csv for Call Automation 

### PR DESCRIPTION
**Pull request by minwoolee-msft**
For some weird reason, ACS Call Automation's python reference is orphaned here: https://learn.microsoft.com/en-us/python/api/azure-communication-callautomation/?view=azure-python

Manually adding entry for Call Automation in python package csv

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -17,6 +17,7 @@
 "azure-communication-phonenumbers","1.1.0","","Communication Phone Numbers","Communication","communication","","","client","true","","03/28/2023","04/26/2021","","","","","","","",""
 "azure-communication-rooms","1.0.0","","Communication Rooms","Communication","communication","","","client","true","","06/06/2023","06/06/2023","","","","","","","",""
 "azure-communication-sms","1.0.1","","Communication Sms","Communication","communication","","","client","true","","06/03/2021","04/20/2021","","","","","","","",""
+"azure-communication-callautomation","1.0.0","","Communication Call Automation","Communication","communication","","","client","true","","06/16/2023","06/16/2023","","","","","","","",""
```
[the remark this answers is not in the document]
Comment (minwoolee-msft) [reply _data/releases/latest/python-packages.csv:20]: This is for GA release - that was already released on June 20 (https://pypi.org/project/azure-communication-callautomation/)

[the remark this answers is not in the document]
Comment (minwoolee-msft) [reply _data/releases/latest/python-packages.csv:20]: Im not sure why, but engineering system pipeline didn't add our python entry correctly

[the remark this answers is not in the document]
Comment (xiangyan99) [reply _data/releases/latest/python-packages.csv:20]: > Im not sure why, but engineering system pipeline didn't add our python entry correctly

This is exactly my question. I don't see your pipeline, neither did you get approval in the API view?

**Review by xiangyan99 — APPROVED** [review]

File: _data/releases/latest/python-packages.csv [hunk 2]
```diff
@@ -16,7 +16,8 @@
 "azure-communication-networktraversal","","1.1.0b1","Communication Network Traversal","Communication","communication","","","client","true","","","","","","","","","","",""
 "azure-communication-phonenumbers","1.1.0","1.2.0b1","Communication Phone Numbers","Communication","communication","","","client","true","","","04/26/2021","","","","","","","",""
 "azure-communication-rooms","1.0.0","","Communication Rooms","Communication","communication","","","client","true","","06/06/2023","06/06/2023","","","","","","","",""
-"azure-communication-sms","1.0.1","","Communication Sms","Communication","communication","","","client","true","","06/04/2021","04/20/2021","","","","","","","",""
+"azure-communication-sms","1.0.1","","Communication Sms","Communication","communication","","","client","true","","06/03/2021","04/20/2021","","","","","","","",""
```
Comment (xiangyan99) [inline _data/releases/latest/python-packages.csv:19]:
```suggestion
"azure-communication-sms","1.0.1","","Communication Sms","Communication","communication","","","client","true","","06/04/2021","04/20/2021","","","","","","","",""
```

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**